sqrl-server/sqrl-server-vertx-base/src/main/java/com/datasqrl/graphql/SubscriptionConfigurationImpl.java

```diff
@@ -38,27 +39,43 @@
  * fetchers for Kafka and PostgreSQL.
  */
 @Slf4j
-@AllArgsConstructor
 public class SubscriptionConfigurationImpl implements SubscriptionConfiguration<DataFetcher<?>> {
 
-  Vertx vertx;
-  ServerConfig config;
-  Promise<Void> startPromise;
+  private final Vertx vertx;
+  private final ServerConfig config;
+  private final List<Future<Void>> subscriptionFutures = new ArrayList<>();
+
+  public SubscriptionConfigurationImpl(Vertx vertx, ServerConfig config) {
+    this.vertx = vertx;
+    this.config = config;
+  }
```

Comment:
We can still spare the ctor with Lombok, e.g. `@RequiredArgsConstructor` annotation.